desktop_version/src/editor.cpp

```diff
@@ -3442,15 +3442,21 @@ void editorrender( KeyPoll& key, Graphics& dwgfx, Game& game, mapclass& map, ent
                 if(ed.tiley<28)
                 {
                     if(ed.roomnamehide>0) ed.roomnamehide--;
-                    FillRect(dwgfx.backBuffer, 0,230+ed.roomnamehide,320,10, dwgfx.getRGB(0,0,0));
-                    dwgfx.Print(5,231+ed.roomnamehide,ed.level[ed.levx+(ed.maxwidth*ed.levy)].roomname, 196, 196, 255 - help.glow, true);
                 }
                 else
                 {
                     if(ed.roomnamehide<12) ed.roomnamehide++;
+                }
+                dwgfx.footerrect.y = 230+ed.roomnamehide;
+                if (dwgfx.translucentroomname)
```

Comment:
Should this part be in the branch above where the FillRect+Print used to be?

Reply:
Well, I'm setting the y-position of `dwgfx.footerrect`, because in the branch where `dwgfx.translucentroomname` is true, I use the y-position of that rect when I pass the rect in the fourth argument to `SDL_BlitSurface()` - `SDL_BlitSurface(dwgfx.footerbuffer, NULL, dwgfx.backBuffer, &dwgfx.footerrect);`.

Thinking about it now, though, I could at least move it into the if-branch, so it's clearer. Doing that now.